### ISSUE TITLE
Is Zee5 Addon  working?

### ISSUE THREAD

**Issue by ksadda77** [Vikassm73/AjaykRepo#294]
Is Zee5 Addon working? It's not working for me for the past 2 months. Pls help.

**Comment by technofreak24** [comment]
@ksadda77 > Is Zee5 Addon working? 

Yes **ZEE5 Addon** works in **KODI Nexus v20.5** or lower like **KODI Nexus v20.4, 20.3** etc.
In **KODI Omega v21.0, v21.1** the ZEE5 Addon** works with changes in addon code.

**Comment by technofreak24** [comment]
> @ksadda77 Is Zee5 Addon working? 

Yes **ZEE5 Addon** works in **KODI Nexus v20.5** or lower like **KODI Nexus v20.4, 20.3** etc.
In **KODI Omega v21.0, v21.1** the ZEE5 Addon** works with changes in addon code.

**Comment by technofreak24** [comment]
> @ksadda77 Is Zee5 Addon working? 

Yes **ZEE5 Addon** works in **KODI Nexus v20.5** or lower like **KODI Nexus v20.4, 20.3** etc.
In **KODI Omega v21.0, v21.1** the **ZEE5 Addon** works with changes in addon code.

**Comment by arnabry11** [comment]
> Is Zee5 Addon working? It's not working for me for the past 2 months. Pls help.

its not working in Kodi v21. But it still works in Kodi v20.5

**Comment by zenom141** [comment]
Hello, How to login into zee5 addon, as it is asking password, but now in zee5 there is no option for password, can you provide login with otp option?

**Comment by Vikassm73** [comment]
Right Click zee5 addon. Enter settings. Please enter your mobile number there. It will ask for OTP

**Comment by ksadda77** [comment]
Every 2 days once, I am getting 'Token Expired' error. I need to uninstall & reinstall the addon whenever I get this error. Pls help to avoid this error.
